test/usecases/forgot_password_use_case_test.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:survey_flutter_ic/model/forgot_password_model.dart';
+import 'package:survey_flutter_ic/usecases/base/base_use_case.dart';
+import 'package:survey_flutter_ic/usecases/forgot_password_use_case.dart';
+import 'package:survey_flutter_ic/usecases/params/forgot_password_params.dart';
+import 'package:survey_flutter_ic/usecases/params/forgot_password_user_params.dart';
+
+import '../mocks/generate_mocks.mocks.dart';
+
+void main() {
+  group('ForgotPasswordUseCaseTest', () {
+    late MockAuthRepository mockAuthRepository;
+    late ForgotPasswordUseCase useCase;
+
+    const email = "email";
+
+    setUp(() {
+      mockAuthRepository = MockAuthRepository();
+      useCase = ForgotPasswordUseCase(mockAuthRepository);
+    });
+
+    test('When forgot password success, it returns success result', () async {
```

Comment:
I think the test case's name should be like this for clearer meaning because `forgot password` is not a valid verb to me
```suggestion
    test('When calling forgot password use case  success, it returns success result', () async {
```